INTITA/Modules/LogInScreen/LogInViewController.swift

```diff
@@ -174,9 +177,9 @@ extension LogInViewController: RegisterButtonTableViewCellDelegate {
     
     func didPressLogInButton(_ sender: RegisterButtonTableViewCell) {
         
-        guard let emailCell = tableView.cellForRow(at: IndexPath(row: 2,
+        guard let emailCell = tableView.cellForRow(at: IndexPath(row: 3,
                                                                  section: 0)) as? TextTableViewCell,
-              let passwordCell = tableView.cellForRow(at: IndexPath(row: 4,
+              let passwordCell = tableView.cellForRow(at: IndexPath(row: 5,
```

Comment:
Use enum rawValue approach instead of hardcoded integers

Reply:
I have just done it.